schemas/common/selfservice.schema.json

```diff
@@ -0,0 +1,170 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This file is licensed to you under the Apache License, Version 2.0 (the 'License');",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at http://www.apache.org/licenses/LICENSE-2.0"
+  ],
+
+
+  "meta:notes" : [
+    "This schema is work in progress. So far it has created a self service object with definitions which ",
+    "are mostly uris, however, it has not defined any structure under those properties many of which would",
+    "be a real object structure. There is an open question as to how much of the implementation detail of ",
+    "one mechanism of deploying should be expressed in this schema and how much should be left to the implementation",
+    "At present, almost everything is left to the implementation with the selfservice object describing the ",
+    "concepts and terminology using uris to refernce implementation details where required.  This may not be sufficiently ",
+    "concrete for our purposes. "
+  ],
+  "$schema": "http://json-schema.org/schema#",
+  "$id": "https://ns.adobe.com/xdm/selfservice",
+  "title": "Self Service Environment",
```

Comment:
I think the name is not entirely intuitive. What about "Customer-managed environment"?